services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,19 +42,38 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
+  if(event == NULL) return ERR_CODE_INVALID_ARG;
+  if(thermalMgrQueueHandle == NULL) return ERR_CODE_INVALID_STATE;
+
+  if(xQueueSend(thermalMgrQueueHandle, event, 0)) return ERR_CODE_QUEUE_FULL;
```

Comment:
Explicitly check that the value for xQueueSend is pdPASS or some equivalent instead of having it converted to a boolean implicitly